docs/introduction-to-smart-contracts.rst

```diff
@@ -301,8 +301,9 @@ If two transactions contradict each other, the one that ends up being second wil
 be rejected and not become part of the block.
 
 These blocks form a linear sequence in time and that is where the word "blockchain"
-derives from. Blocks are added to the chain in rather regular intervals - for
-Ethereum this is roughly every 17 seconds.
+derives from. Blocks are added to the chain in rather regular intervals. These intervals
+are subject to change in the future. For the latest information, it is recommended to
+monitor the network.
```

Comment:
@cameel & @nikola-matic Maybe we could help with *where* users can monitor, like https://etherscan.io/chart/blocktime?

For the latest information, it is recommended to
monitor the network for example on [Etherscan](https://etherscan.io/chart/blocktime).

Reply:
Thanks for the feedback. I've updated the documentation.